packages/react/test/specs/components/Tree/Tree-test.tsx

```diff
@@ -156,5 +156,23 @@ describe('Tree', () => {
         .simulate('keydown', { keyCode: keyboardKey['*'] })
       checkOpenTitles(wrapper, ['1', '11', '12', '2', '21', '22', '3'])
     })
+
+    it('should have items expanded based on their expanded prop', () => {
```

Comment:
Maybe it would be worther to add tests for expand on second/third item. This doesn't work currently:
```
import React from 'react'
import { Tree } from '@stardust-ui/react'

const items = [
  {
    id: '1',
    title: 'House Lannister',
    items: [
      {
        id: '11',
        title: 'Tywin',
        items: [
          {
            id: '111',
            title: 'Jaime',
          },
          {
            id: '112',
            title: 'Cersei',
          },
          {
            id: '113',
            title: 'Tyrion',
          },
        ],
      },
      {
        id: '12',
        title: 'Kevan',
        items: [
          {
            id: '121',
            title: 'Lancel',
          },
          {
            id: '122',
            title: 'Willem',
          },
          {
            id: '123',
            title: 'Martyn',
          },
        ],
      },
    ],
  },
  {
    id: '2',
    title: 'House Targaryen',
    expanded: true,
    items: [
      {
        id: '21',
        title: 'Aerys',
        items: [
          {
            id: '211',
            title: 'Rhaegar',
          },
          {
            id: '212',
            title: 'Viserys',
          },
          {
            id: '213',
            title: 'Daenerys',
          },
        ],
      },
    ],
  },
]

const TreeInitiallyOpenExampleShorthand = () => (
  <Tree items={items} aria-label="Initially open" />
)

export default TreeInitiallyOpenExampleShorthand

```

See https://github.com/microsoft/fluent-ui-react/pull/2061/files#r344072359 for possible fix

Reply:
are you sure this does not work? If I have `expanded` on  `Tywin` and open `House Lannister` manually, we find `Tywin` already expanded.

Reply:
I would expect `House Targaryen` to be opened, but it is not, so I guess it doesn't work